source/incident-collaboration/setting-up-playbooks.rst

```diff
@@ -60,12 +60,12 @@ When a run starts
   * **Invite members**: Enable the toggle, and select members who will automatically be invited to the channel when the playbook run starts.
   * **Assign the owner role**: Enable the toggle and select a member. This member is automatically assigned as the owner of the playbook run.
   * **Announce in another channel**: Enable the toggle and select a channel. When the playbook run is started, an announcement is made in the selected channel.
-  * **Send outgoing webhook**: Enable the toggle and enter the webhook you want to use for when the run starts.
+  * **Send outgoing webhook**: Enable the toggle and enter the webhook you want to use for when the run starts. For technical information about webhooks, please see the `outgoing webhooks <https://developers.mattermost.com/integrate/outgoing-webhooks/>`_ page. 
```

Comment:
Content need to be added on that linked page for outgoing playbook run webhook

Reply:
Right, I'm with you. 

I can update that page with the payload information from @agarciamontoro if it'll fit there. I don't see why it wouldn't fit in a new section, with the content added. @hanzei, would you have any issues with the playbooks webhook payload content being added to https://developers.mattermost.com/integrate/outgoing-webhooks?

Reply:
The change would be a new section called Playbooks webhooks, with the payload information @agarciamontoro provided.

Reply:
We can either have this information in the core docs, or have some similar documentation ourselves. I was working on a better API documentation for Playbooks (something similar to api.mattermost.com), but it's not yet finished. That would be a great place to put this information.

Reply:
@agarciamontoro, as discussed, let's link to the code for now as it is the most current source of this information and with the addition of comments it will be very helpful. :)